numpy/core/_add_newdocs.py

```diff
@@ -5616,12 +5618,12 @@ def luf(lamdaexpr, *args, **kwargs):
     second an int32:
 
     >>> np.dtype([('f1', np.uint), ('f2', np.int32)])
-    dtype([('f1', '<u4'), ('f2', '<i4')])
+    dtype([('f1', '<u8'), ('f2', '<i4')])
```

Comment:
This isn't going to help if we switch back to linux. Use a sized type instead.

Reply:
I've tried to enforce sized types where I observed issues like this throughout.

Reply:
I switched to a sized unsigned integer type so marking as resolved